### ISSUE TITLE
Add support for vertical slider to NumberControl

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/scenery-phet#836]
For https://github.com/phetsims/faradays-electromagnetic-lab/issues/61, I need 2 NumberControl instances for the AC Power Supply.  One of them needs to be a vertical slider, see screenshot below.   Unfortunately, NumberControl does not currently support vertical sliders, because of this:

```ts
428     this.slider = new HSlider( numberProperty, numberRange, options.sliderOptions );
```

Fingers crossed that this is a simple matter of changing HSlider to Slider.

<img width="289" alt="screenshot_2996" src="https://github.com/phetsims/scenery-phet/assets/3046552/e9019a78-5c72-4573-b3a9-ac0c3a035622">

**Comment by pixelzoom** [comment]
Done in the above commit.  I also added a vertical NumberControl to the scenery-phet demo, see screenshot below.

I did NOT add the ability to change the arrow buttons to point up/down, something that might be needed. I don't need that for https://github.com/phetsims/faradays-electromagnetic-lab/issues/61.  So since there's no actual use for it yet, the person who needs it will need to add that feature. 

Random selection chose @zepumph for a review. If you don't have time, assign back to me and I'll choose someone else.

<img width="944" alt="screenshot_2998" src="https://github.com/phetsims/scenery-phet/assets/3046552/aa31b19f-f220-4ae5-b839-b2607c6f191c">
